### PR TITLE
prow/plugins: enable milestone plugins for nfd

### PR DESCRIPTION
**Pull request by marquiz**
Enable the milestone command for the
kubernetes-sigs/node-feature-discovery project.

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -1410,6 +1413,7 @@ plugins:
 
   kubernetes-sigs/node-feature-discovery:
     plugins:
+    - milestone
```
Comment (ArangoGutierrez) [inline config/prow/plugins.yaml:1416]: Why not 
```yaml
    - milestone
    - milestonestatus
```
?

Comment (marquiz) [reply config/prow/plugins.yaml:1416]: Yes, why not. Added...

**Comment by marquiz** [comment]
/retitle prow/plugins: enable milestone plugins for nfd

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Review by fmuyassarov — APPROVED** [review]
/lgtm

**Comment by ArangoGutierrez** [comment]
PING @cblecker for approval

**Pull request merged**